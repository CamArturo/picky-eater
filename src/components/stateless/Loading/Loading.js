import React, { Component } from 'react';
import loadingGif from '../../../assets/loader.gif'
import PropTypes from 'prop-types';
import '../Cuisine/Cuisine.css';

class Loading extends Component {
  render () {
    return (
      <article className='loading-gif'>
        <img src={loadingGif} alt="" />
      </article>
    );
  }
}

Loading.propTypes = {};

export default Loading;