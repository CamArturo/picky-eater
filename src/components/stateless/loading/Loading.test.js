import React from 'react';
import { shallow } from 'enzyme';
import Loading from './Loading';


it('Should match the snapshot', () => {
  const appComponent = shallow(<Loading />);

  expect(appComponent).toMatchSnapshot();
});