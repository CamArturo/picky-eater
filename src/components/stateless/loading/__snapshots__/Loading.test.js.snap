// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Loading />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <img
        alt="Loading"
        src="loader.gif"
/>,
      "className": "loading-gif",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "alt": "Loading",
        "src": "loader.gif",
      },
      "ref": null,
      "rendered": null,
      "type": "img",
    },
    "type": "article",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <img
          alt="Loading"
          src="loader.gif"
/>,
        "className": "loading-gif",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "Loading",
          "src": "loader.gif",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      "type": "article",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
